src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,26 @@
+import { space, SpacingUnit } from "@artsy/palette"
+import styled from "styled-components"
+import { media } from "../../Components/Helpers"
```

Comment:
This refers to the old media helper (which we need to add a deprecation note about so that people don't use it in new framework code). The proper import is 
```
import { media } from 'Styleguide/Elements/Grid'
```
(and make sure the path is absolute)